### PR TITLE
Feature/sl 1057 perf tests in pipeline

### PR DESCRIPTION
**Pull request by kpakur**
https://tools.hmcts.net/jira/browse/SL-1057

### CODE REVIEW

File: src/gatling/simulations/uk/gov/hmcts/reform/sandl/snlrules/performance/NewHearingPartSimulation.scala [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  setUp(
+      scn.inject(
+        rampUsers(10) over(2 seconds),
+      )
+    )
+    .maxDuration(15 seconds)
+    .protocols(httpConf)
+    .assertions(
+      global.failedRequests.count.is(0)
```
Comment (satyachundur) [inline src/gatling/simulations/uk/gov/hmcts/reform/sandl/snlrules/performance/NewHearingPartSimulation.scala]: Are we not worried about the response times verification i.e., to make sure response times are not greater than 2 sec?

File: src/gatling/simulations/uk/gov/hmcts/reform/sandl/snlrules/performance/NewSessionSimulation.scala [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  setUp(
+    scn.inject(
+      rampUsers(10) over(2 seconds),
+    )
+  )
+    .maxDuration(15 seconds)
+    .protocols(httpConf)
+    .assertions(
+      global.failedRequests.count.is(0)
```
Comment (satyachundur) [inline src/gatling/simulations/uk/gov/hmcts/reform/sandl/snlrules/performance/NewSessionSimulation.scala]: Same as above? assertions on response times?

File: src/gatling/simulations/uk/gov/hmcts/reform/sandl/snlrules/performance/NewSessionSimulation.scala [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.sandl.snlrules.performance
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import scala.concurrent.duration._
+
+class NewSessionSimulation extends Simulation {
+  val httpConf = http
+    .baseURL(Environments.baseUrl)
```
Comment (satyachundur) [inline src/gatling/simulations/uk/gov/hmcts/reform/sandl/snlrules/performance/NewSessionSimulation.scala]: Base url seems to be not using ignore certificates like "acceptAnyCertificate = true  "  will that work is the PR does not have valid certificate?

File: src/gatling/simulations/uk/gov/hmcts/reform/sandl/snlrules/performance/NewHearingPartSimulation.scala (hunk 1 above)
Comment (kpakur) [reply src/gatling/simulations/uk/gov/hmcts/reform/sandl/snlrules/performance/NewHearingPartSimulation.scala]: this file is removed now, but added the mentioned limit to 2 seconds on the PipelineSimulation

File: src/gatling/simulations/uk/gov/hmcts/reform/sandl/snlrules/performance/NewSessionSimulation.scala (hunk 3 above)
Comment (kpakur) [reply src/gatling/simulations/uk/gov/hmcts/reform/sandl/snlrules/performance/NewSessionSimulation.scala]: uncommented the option in conf file and set to true

File: src/gatling/simulations/uk/gov/hmcts/reform/sandl/snlrules/performance/NewSessionSimulation.scala (hunk 2 above)
Comment (kpakur) [reply src/gatling/simulations/uk/gov/hmcts/reform/sandl/snlrules/performance/NewSessionSimulation.scala]: as above

**Pull request merged**